browse-scaler/src/index.js

```diff
@@ -35,6 +48,15 @@ const getImageUrlFromConcept = async (conceptId, conceptType) => {
   return null;
 };
 
+/**
+ * resizeImageFromConceptId: call necessary helper functions to ultimately
+ * result in the response that the server will send
```

Comment:
This comment doesn't really say what this does, which, I imagine, is resize an image associated with a given concept-id.

Reply:
yup - fixed